src/main/resources/static/js/subscription.js

```diff
@@ -459,13 +466,17 @@ jQuery(document).ready(function () {
                     "title": "Action",
                     "data": null,
                     "render": function (data, type, row, meta) {
-
-                        if (isSecured == false || (row.ldapUserName == currentUser && row.ldapUserName != null)) {
+                        var unsecureEditAll = isSecured == false;
+                        var securedEditOwn = row.ldapUserName == currentUser && row.ldapUserName != null;
+                        var allUserEditNoOwner = row.ldapUserName.length == 0 && currentUser != undefined;
+                        if (unsecureEditAll || securedEditOwn || allUserEditNoOwner) {
                             return '<button id="view-' + data.subscriptionName + '" class="btn btn-sm btn-success view_record table-btn">View</button> '
                                 + '<button id="edit-' + data.subscriptionName + '" class="btn btn-sm btn-primary edit_record table-btn">Edit</button> '
                                 + '<button id="delete-' + data.subscriptionName + '" class="btn btn-sm btn-danger delete_record table-btn">Delete</button>';
                         } else {
-                            return '<button id="view-' + data.subscriptionName + '" class="btn btn-sm btn-success view_record table-btn">View</button>';
+                            return '<button id="view-' + data.subscriptionName + '" class="btn btn-sm btn-success view_record table-btn">View</button> '
```

Comment:
The only difference with those 2 returns is the (disabled="") Maybe this could be rewritten to not have duplicated code? Maybe the button forming should be in it´s own function, or function bound to a variable.